server/api/core/accounts/password.js

```diff
@@ -231,3 +231,118 @@ export function sendVerificationEmail(userId, email) {
     html: SSR.render(tpl, dataForEmail)
   });
 }
+
+
+/**
+ * Send an email with a link the user can use to verify their updated email address.
+ * @param {String} userId - The id of the user to send email to.
+ * @param {String} [email] Optional. Address to send the email to.
+ *                 This address must be in the user's emails list.
+ *                 Defaults to the first unverified email in the list.
+ * @return {Job} - returns a sendEmail Job instance
+ */
+export function sendUpdatedVerificationEmail(userId, email) {
+  // Make sure the user exists, and email is one of their addresses.
+  const user = Meteor.users.findOne(userId);
+
+  if (!user) {
+    Logger.error("sendVerificationEmail - User not found");
+    throw new Meteor.Error("user-not-found", "User not found");
+  }
+
+  let address = email;
+
+  // pick the first unverified address if no address provided.
+  if (!email) {
+    const unverifiedEmail = _.find(user.emails || [], (e) => !e.verified) || {};
+
+    address = unverifiedEmail.address;
+
+    if (!address) {
+      const msg = "No unverified email addresses found.";
+      Logger.error(msg);
+      throw new Meteor.Error("no-unverified-address", msg);
+    }
+  }
+
+  // make sure we have a valid address
+  if (!address || !_.includes(_.map(user.emails || [], "address"), address)) {
```

Comment:
`_.includes` should be `Array.includes` (I know you likely copied this from the other email verification function that I wrote.  Just trying to be do it right moving forward. :))